examples/__tests__/test-utils.ts

```diff
@@ -0,0 +1,7 @@
+import { basename, join } from 'path';
+import { testDeployment } from '../../test/lib/deployment/test-deployment.js';
+
+export async function deployExample(filename: string) {
+  const example = basename(filename).slice(0, -8);
```

Comment:
There's something about slicing that scares me. I don't immediately know what is being sliced.

```suggestion
  const example = basename(filename).replace(/\.test\.ts$/, '');
```

Reply:
Done [862b1fa](https://github.com/vercel/vercel/pull/9403/commits/862b1fa6efa3ec5916d4b5a6f807977018fe94a8)